openedx/features/course_search/.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  extends: 'eslint-config-edx',
+  root: true,
+  settings: {
+    'import/resolver': {
+      webpack: {
+        config: 'webpack.dev.config.js',
```

Comment:
course_search isn't using webpack yet.  But this solves some of the import errors eslint was throwing, by letting eslint figure out how our code is layed out.